include/miopen/miopen.h

```diff
@@ -5793,6 +5794,37 @@ MIOPEN_EXPORT miopenStatus_t miopenSumForward(miopenHandle_t handle,
 // CLOSEOUT SUM DOXYGEN GROUP
 #endif
 
+#ifdef MIOPEN_BETA_API
+// Padreflection APIs
+/** @addtogroup padreflection
+ *
+ *  @{
+ */
+/*! @brief Add padding by the reflection of the tensor
+ *
+ * @param handle                   MIOpen handle (input)
```

Comment:
I think you missed this part, commenting.

Reply:
Resolved in commit https://github.com/moreh-dev/MIOpen/pull/7/commits/6f4e1a0352dfe9444be3f7b7dd23b925ae5e2491